core-services/prow/02_config/openshift/cloud-provider-alibaba-cloud/_pluginconfig.yaml

```diff
@@ -0,0 +1,13 @@
+approve:
```

Comment:
I thought this used to all go in one big file? Has this changed?

Reply:
it was generated by `make new-repo`.